Backoffice/Form/Type/FieldTypeType.php

```diff
@@ -74,21 +89,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label' => 'open_orchestra_backoffice.form.field_type.orderDirection',
                 'required' => false,
             ))
-            ->add('translatable', 'checkbox', array(
-                'label' => 'open_orchestra_backoffice.form.field_type.translatable',
-                'required' => false,
-            ))
-            ->add('type', 'choice', array(
-                'choices' => $this->getChoicesType(),
-                'label' => 'open_orchestra_backoffice.form.field_type.type',
-                'attr' => array(
-                    'class' => 'content_type_change_type'
-                )
-            ))
-            ->add('listable', 'checkbox', array(
-                'label' => 'open_orchestra_backoffice.form.field_type.listable',
+            ->add('additionalParameters', "text", array(
```

Comment:
Not the right way to do this. You should use twig to add <hr />. This formType is named oo_field_type. So you can add a twig with :
{% block oo_field_type %}
    {% spaceless %}
    	{{ form_start() }}
    	{{ form_row(form.fieldId) }}
    	{{ form_row(form.labels) }}
    	...
        <hr />
        {{ form_rest() }}
        {{ form_end() }}
    {% endspaceless %}
{% endblock %}

not totally sure for form_rest and form_end. I let you check.